idea/testData/quickfix/addRunBeforeLambda/basic.kt.after

```diff
@@ -0,0 +1,6 @@
+// "Add 'run' before the lambda expression" "true"
+// ERROR: Unresolved reference: run
```

Comment:
You can add WITH_RUNTIME directive so that the `run` function will be resolved correctly.